### PR TITLE
bitcoin-computer__lib types update

### PR DESCRIPTION
**Pull request by jonty007**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] tested lib in browser
- [X] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Bitcoin Computer](https://docs.bitcoincomputer.io/api/)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/bitcoin-computer__lib/package.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+    "owners": [
+      {"name": "Clemens Ley", "githubUsername": "ClemensLey"},
+      {"name": "Laura Tardivo", "githubUsername": "ltardivo"},
+      {"name": "Vivek Singh", "githubUsername": "jonty007"}
     ],
+    "private": true,
+    "projects": ["https://docs.bitcoincomputer.io", "https://www.bitcoincomputer.io"],
     "devDependencies": {
-        "@types/bitcoin-computer__lib": "workspace:."
+      "@types/bitcoin-computer__lib": "workspace:."
```
Comment (codershiba) [inline types/bitcoin-computer__lib/package.json:12]:
```suggestion
      "@types/bitcoin-computer__lib": "workspace:.",
      "@types/node": "*"
```

File: types/bitcoin-computer__lib/tsconfig.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
         "noImplicitThis": true,
         "strictFunctionTypes": true,
         "strictNullChecks": true,
-        "types": [],
+        "types": ["node"],
```
Comment (codershiba) [inline types/bitcoin-computer__lib/tsconfig.json:11]:
```suggestion
        "types": [],
```

File: types/bitcoin-computer__lib/package.json [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+      {"name": "Clemens Ley", "githubUsername": "ClemensLey"},
+      {"name": "Laura Tardivo", "githubUsername": "ltardivo"},
+      {"name": "Vivek Singh", "githubUsername": "jonty007"}
     ],
+    "private": true,
+    "projects": ["https://docs.bitcoincomputer.io", "https://www.bitcoincomputer.io"],
     "devDependencies": {
-        "@types/bitcoin-computer__lib": "workspace:."
+      "@types/bitcoin-computer__lib": "workspace:.",
+      "@types/node": "*"
```
Comment (codershiba) [inline types/bitcoin-computer__lib/package.json:13]:
```suggestion
```

File: types/bitcoin-computer__lib/package.json [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
     },
-    "owners": [
-        {
-            "name": "jonty007",
-            "githubUsername": "jonty007"
-        }
-    ]
-}
+    "dependencies": {}
```
Comment (codershiba) [inline types/bitcoin-computer__lib/package.json:15]:
```suggestion
    "dependencies": {
        "@types/node": "*"
    }
```

**Review by ghost — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

**Comment by jonty007** [comment]
Ready to merge

**Pull request merged**